config/jobs/image-pushing/k8s-staging-external-dns.yaml

```diff
@@ -9,6 +9,8 @@ postsubmits:
       decorate: true
       branches:
         - ^master$
+        # this is a regex for semver, from https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
+        - ^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$
```

Comment:
Do we need to be so correct? Seems like `v[0-9].*` will work just as well and is far more readable

Reply:
Not really. I simplified it to your suggestion.